legacy/core/src/main/java/com/fsck/k9/Preferences.kt

```diff
@@ -77,7 +79,7 @@ class Preferences internal constructor(
             val accountUuids = storage.getString("accountUuids", null)
             if (!accountUuids.isNullOrEmpty()) {
                 accountUuids.split(",").forEach { uuid ->
-                    val newAccount = Account(uuid)
+                    val newAccount = Account(uuid) { K9.isSensitiveDebugLoggingEnabled }
```

Comment:
The trailing lambda syntax looks odd to me here. I think the following would be easier to read:

```suggestion
                    val newAccount = Account(uuid, K9::isSensitiveDebugLoggingEnabled)
```